Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm

```diff
@@ -165,6 +176,29 @@ void didReceiveAuthenticationChallenge(Ref<WebKit::AuthenticationChallengeProxy>
 
         [m_delegate.getAutoreleased() didReceiveAuthenticationChallenge:nsURLChallenge completionHandler:completionHandler.get()];
     }
+#if ENABLE(SCREEN_TIME)
+    void getScreenTimeURLs(const String& identifer, CompletionHandler<void(HashSet<URL>&&)>&& callback) const final
+    {
+        if (![PAL::getSTWebHistoryClass() instancesRespondToSelector:@selector(fetchAllHistoryWithCompletionHandler:)])
+            return;
+
+        // Create STWebHistory
+        STWebHistoryProfileIdentifier const WebHistoryProfileIdentifier = identifer;
+        RetainPtr webHistory = adoptNS([PAL::allocSTWebHistoryInstance() initWithProfileIdentifier:WebHistoryProfileIdentifier]);
+
+        // Use STWebHistory to fetch all website history for identifier
+        [webHistory fetchAllHistoryWithCompletionHandler:^(NSSet<NSURL *> *urls, NSError *  error) {
+            UNUSED_PARAM(error);
```

Comment:
I think we wouldn't surface an unused param warning here anyway, so maybe safe to just remove this? :)